spec/models/petition_spec.rb

```diff
@@ -3022,4 +3122,58 @@
       I18n.with_locale(:"cy-GB") { expect(Petition.new.locale).to eq("cy-GB") }
     end
   end
+
+  describe "#create" do
+    it "sets threshold for referral to Site value" do
+      petition = FactoryBot.build(:petition)
+
+      expect {
+        petition.save
+      }.to change {
+        petition.read_attribute(:threshold_for_referral)
+      }.from(nil).to(Site.threshold_for_referral)
+    end
+
+    it "sets threshold for debate to Site value" do
+      petition = FactoryBot.build(:petition)
+
+      expect {
+        petition.save
+      }.to change {
+        petition.read_attribute(:threshold_for_debate)
+      }.from(nil).to(Site.threshold_for_debate)
+    end
+  end
+
+  describe "#threshold_for_referral" do
+    it "returns petition value if present" do
+      petition = FactoryBot.create(:petition)
+      petition.update(threshold_for_referral: 50)
```

Comment:
There's no need to call `create` here - you can pass in the value to `build` and the callback won't have overridden it. Also saves two trips to the database.